Installation on Windows via GitHub action fails

This used to work until last week... but now installing Nim on Windows (using GitHub actions) via choosenim fails:

```
2024-10-27T11:03:52.9527486Z ##[group]Run iffy/install-nim@v4.5.0
2024-10-27T11:03:52.9527978Z with:
2024-10-27T11:03:52.9528241Z   version: stable
2024-10-27T11:03:52.9528529Z env:
2024-10-27T11:03:52.9528789Z   CHOOSENIM_CHOOSE_VERSION: stable
2024-10-27T11:03:52.9529171Z   CHOOSENIM_NO_ANALYTICS: 1
2024-10-27T11:03:52.9529507Z   MSYSTEM: MINGW64
2024-10-27T11:03:52.9529779Z ##[endgroup]
2024-10-27T11:03:52.9797562Z ##[group]Run export SHELL=/bin/bash
2024-10-27T11:03:52.9797906Z [36;1mexport SHELL=/bin/bash[0m
2024-10-27T11:03:52.9798362Z [36;1m'D:\a\_actions\iffy\install-nim\v4.5.0/install-nim.sh' 'stable'[0m
2024-10-27T11:03:52.9811566Z shell: C:\Program Files\Git\bin\bash.EXE --noprofile --norc -e -o pipefail {0}
2024-10-27T11:03:52.9812079Z env:
2024-10-27T11:03:52.9812311Z   CHOOSENIM_CHOOSE_VERSION: stable
2024-10-27T11:03:52.9812623Z   CHOOSENIM_NO_ANALYTICS: 1
2024-10-27T11:03:52.9812890Z   MSYSTEM: MINGW64
2024-10-27T11:03:52.9813115Z ##[endgroup]
2024-10-27T11:03:53.7130224Z Install type: choosenim
2024-10-27T11:03:53.7130849Z        param: stable
2024-10-27T11:03:53.7188676Z Installing via choosenim for: stable
2024-10-27T11:03:53.8296172Z Found version 
2024-10-27T11:03:54.9059805Z choosenim-init: Downloading choosenim-0.8.9_windows_amd64.zip
2024-10-27T11:03:55.7524090Z main: line 79: /tmp/choosenim-0.8.9_windows_amd64.zip: cannot execute binary file: Exec format error
2024-10-27T11:03:55.7604769Z ##[error]Process completed with exit code 126.
```

See:
- [Failed run with iffy/install-nim](https://github.com/h3rald/min/actions/runs/11539919663/job/32120145575#step:9:1)
- [Failed run with my own action](https://github.com/h3rald/min/actions/runs/11534457519/job/32108530206)
- [Successful run with my own action](https://github.com/h3rald/min/actions/runs/11418786584/job/31772634009)

--- a/.github/workflows/test_macos_arm64.yml
+++ b/.github/workflows/test_macos_arm64.yml
@@ -1,0 +1,21 @@
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os:
+          - windows-latest
+          - macos-latest
+          - ubuntu-latest
+        nimversion:
+          - devel
+          # - stable
+    name: ${{ matrix.os }} - ${{ matrix.nimversion }}
+    runs-on: ${{ matrix.os }}
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - run: curl https://raw.githubusercontent.com/nim-lang/choosenim/refs/heads/master/scripts/choosenim-unix-init.sh -sSf | sh

--- a/.github/workflows/test_macos_arm64.yml
+++ b/.github/workflows/test_macos_arm64.yml
@@ -6,7 +6,10 @@ jobs:
     strategy:
       matrix:
         os:
+          - windows-latest
+          - macos-13
           - macos-latest
+          - ubuntu-latest
         nimversion:
           - devel
           # - stable

--- a/.github/workflows/test_macos_arm64.yml
+++ b/.github/workflows/test_macos_arm64.yml
@@ -18,4 +18,4 @@ jobs:
     env:
       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     steps:
-      - run: curl https://raw.githubusercontent.com/nim-lang/choosenim/refs/heads/master/scripts/choosenim-unix-init.sh -sSf -y | sh -y
+      - run: curl https://raw.githubusercontent.com/nim-lang/choosenim/refs/heads/master/scripts/choosenim-unix-init.sh -sSf | sh -y

--- a/.github/workflows/test_macos_arm64.yml
+++ b/.github/workflows/test_macos_arm64.yml
@@ -18,4 +18,4 @@ jobs:
     env:
       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     steps:
-      - run: curl https://raw.githubusercontent.com/nim-lang/choosenim/refs/heads/master/scripts/choosenim-unix-init.sh -sSf | sh
+      - run: curl https://raw.githubusercontent.com/nim-lang/choosenim/refs/heads/master/scripts/choosenim-unix-init.sh -sSf -y | sh -y

--- a/.github/workflows/test_macos_arm64.yml
+++ b/.github/workflows/test_macos_arm64.yml
@@ -18,4 +18,4 @@ jobs:
       - name: choosenim
         run: |
           curl https://raw.githubusercontent.com/nim-lang/choosenim/refs/heads/master/scripts/choosenim-unix-init.sh -o init.sh
-          sh init.sh
+          sh init.sh -y

--- a/.github/workflows/test_macos_arm64.yml
+++ b/.github/workflows/test_macos_arm64.yml
@@ -1,5 +1,4 @@
 on:
-  push:
   pull_request:
 
 jobs:
@@ -7,9 +6,7 @@ jobs:
     strategy:
       matrix:
         os:
-          - windows-latest
           - macos-latest
-          - ubuntu-latest
         nimversion:
           - devel
           # - stable
@@ -18,4 +15,7 @@ jobs:
     env:
       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     steps:
-      - run: curl https://raw.githubusercontent.com/nim-lang/choosenim/refs/heads/master/scripts/choosenim-unix-init.sh -sSf | sh -y
+      - name: choosenim
+        run: |
+          curl https://raw.githubusercontent.com/nim-lang/choosenim/refs/heads/master/scripts/choosenim-unix-init.sh -o init.sh
+          sh init.sh

